DataStructuresMemories/DataStructuresMemoriesTableViewController.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  DataStructuresMemoriesTableViewController.swift
+//  DataStructuresMemories
+//
+//  Created by Oleksii Kolomiiets on 4/12/18.
+//  Copyright © 2018 Oleksii Kolomiiets. All rights reserved.
+//
+
+import UIKit
+
+
+
+class DataStructuresMemoriesTableViewController: UITableViewController {
+   
+    let dataStructModel = DataStructModel()
+    
+    // MARK: - UiTableViewController
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        tableView.separatorColor = #colorLiteral(red: 0.1215686275, green: 0.1294117647, blue: 0.1411764706, alpha: 1)
```

Comment:
Easy, Easy, bro! 
Why so precious?!

Reply:
it is long story)) But I've already fixed that and added method for changing the color for separator